docs/overview.md

```diff
@@ -25,14 +25,14 @@ Typically, your component will be made up of the following pieces:
 - Your component code
 - An implementation of the 'BotComponent' class to register your components with the runtime
   - You'll also need to update the `components` array in the `appsettings.json` file with your component name
-- A schema file that declares the inputs and outputs your component requires
-- A `uischema` file to tell Composer where to display your components
+- One or more `.schema` files that declares the inputs and outputs your component requires
+- One or more `.uischema` files to tell Composer where to display your components
```

Comment:
uischema is optional, right?  But recommended?

Reply:
This is just the overview, we'd go into all of those details in the "how to create a package" article.